Desktop/main.cpp

```diff
@@ -461,7 +461,8 @@ int main(int argc, char *argv[])
 			QmlUtils::configureQMLCacheDir();
 		#endif
 
-			QLocale::setDefault(QLocale(QLocale::English)); // make decimal points == .
+			if(Settings::value(Settings::USE_ALT_LOCALE).toBool())
+				QLocale::setDefault(QLocale(QLocale::English)); // make decimal points == .
```

Comment:
Why do we need that here?

Reply:
we dont, it is still there from when the option was called "use_usa_locale"